restlight-server/src/main/java/esa/restlight/server/BaseDeployments.java

```diff
@@ -73,14 +73,13 @@
     private RestlightHandler handler;
 
     static {
-        final ThreadPoolExecutor e = new ThreadPoolExecutor(0,
-                1,
-                0L,
-                TimeUnit.SECONDS,
-                new LinkedBlockingQueue<>(),
-                new RestlightThreadFactory("useless"));
-        JDK_DEFAULT_REJECT_HANDLER = e.getRejectedExecutionHandler().getClass();
-        e.shutdownNow();
+        Class<? extends RejectedExecutionHandler> defaultHandlerClass = ThreadPoolExecutor.AbortPolicy.class;
+        try {
+            defaultHandlerClass = (Class<? extends RejectedExecutionHandler>) ThreadPoolExecutor.class.getDeclaredField("defaultHandler").getType();
+        } catch (NoSuchFieldException e) {
+            LoggerUtils.logger().error("Could not find the field named defaultHandler in ThreadPoolExecutor, JDK_DEFAULT_REJECT_HANDLER will be set to ThreadPoolExecutor$AbortPolicy.class", e);
```

Comment:
What about printing the log by debug-level and fallbacking to `ThreadPoolExecutor.getRejectedExecutionHandler().getClass()` if failed to retrieve `JDK_DEFAULT_REJECT_HANDLER` by reflection